README.md

```diff
@@ -117,21 +117,6 @@ logdna register <email>
 # now paste the Ingestion Key into the kubectl commands above
 ```
 
-## Kubernetes Logging
```

Comment:
@darinspivey, I'm still not 100% sure whether it's good idea to remove this whole block since we just stop building new Docker images and there are still `yamls` inside this repo. Plus, how consistent is this with our other docs for `k8s`? Should we have separate PR for this?

Reply:
I think it's *really* confusing to leave stuff in (although I'm fine with keeping the yamls).  If this repo no longer does k8s for v1, we shouldn't keep saying that it does.  I think we're covered by the `major` change for this.

Reply:
`./docs` should be enough to cover that side of the story - so, nothing to worry